packages/carbon-web-components/src/components/code-snippet/code-snippet.scss

```diff
@@ -93,12 +93,17 @@ $css--plex: true !default;
     }
   }
 
+  .#{$prefix}--snippet-button {
+    z-index: 1;
+  }
+
   .#{$prefix}--snippet-container {
     display: block;
     overflow: hidden;
     position: relative;
     height: auto;
     padding-bottom: 0;
+    padding-right: $carbon--spacing-07;
```

Comment:
Wondering if this should be `1.5rem`, such that the code buts right up against the copy button, which is how the single line code snippet is currently:

![image](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/191049/5aed9319-6e2e-48dc-8f37-261c5e8ca3e0)

Here is how it is with this change:

![image](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/191049/641ec387-7cdc-4424-8eb6-953b4340d319)

Reply:
Although I could see the argument going for the 0.5rem spacing all around the copy button 🤷. Either way, I'm good with it 😀.

Reply:
I like that suggestion! Use as little space as possible while still allowing the text to clear the button once scrolled all the way to the right.